packages/cashier/src/modules/deposit-fiat/components/deposit-fiat-iframe/deposit-fiat-iframe.scss

```diff
@@ -0,0 +1,9 @@
+.deposit-fiat-iframe {
+    &__iframe {
+        width: 100%;
+        height: 75vh;
+    }
+    &__loader {
+        height: 75vh;
+    }
```

Comment:
```suggestion
```
@aum-deriv No need to add style for it, Instead please add `min-height: 100%` to `&__content` class in `packages/cashier/src/components/page-container/page-container.scss`